src/utils/isUrl.ts

```diff
@@ -3,14 +3,11 @@
  * @param url - string to check.
  */
 export const isUrl = (url: string): boolean => {
-  const pattern = new RegExp(
-    '^(http://|https://)[a-z0-9]+([-.]{1}[a-z0-9]+)*.[a-z]{2,5}(:[0-9]{1,5})?(/.*)?$',
-    'gi'
-  )
+  try {
```

Comment:
Might be a good idea to add tests for this function.

Reply:
test is provided